gazelle/buf/kinds.go

```diff
@@ -55,6 +56,20 @@ var bufKinds = map[string]rule.KindInfo{
 			"modules": true,
 		},
 	},
+	pushRuleKind: {
+		MatchAttrs: []string{"name"},
```

Comment:
```
breakingRuleKind: {
		MatchAttrs: []string{"targets"},
```
Since these rules treat targets the same way and share code related to resolving, should their matchattrs be the same?